### PR TITLE
Add Script Command: github-unread-notifications

### PR DESCRIPTION
**Pull request by crstauf**

### CODE REVIEW

**Comment by thomaspaulmann** [comment]
Top addition! I'll favorite that command in Raycast to see upcoming PRs from you ;-)

File: developer-utils/github-unread-notifications.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# @raycast.title GitHub Unread Notifications
+# @raycast.mode inline
+# @raycast.refreshTime 5m
+
+# Optional parameters:
+# @raycast.author Caleb Stauffer
+# @raycast.authorURL https://github.com/crstauf
+# @raycast.description Number of unread GitHub notifications.
+# @raycast.packageName GitHub
+# @raycast.icon images/github-logo.png
```
Comment (thomaspaulmann) [inline developer-utils/github-unread-notifications.sh:24]: Reminds me that we should have support for dark and light theme icons.

File: developer-utils/github-unread-notifications.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+# GitHub username
+# user=
+
+# GitHub personal access token
+# access_key=
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title GitHub Unread Notifications
```
Comment (thomaspaulmann) [inline developer-utils/github-unread-notifications.sh:15]:
```suggestion
# @raycast.title Unread Notifications
```

We usually don't specify the extension name in the command, e.g. `My Open Pull Requests`.

File: developer-utils/github-unread-notifications.sh [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	echo "jq is required (https://stedolan.github.io/jq/).";
+	exit 1;
+fi
+
+auth=$(echo -n "$user:$access_key" | base64)
+response=$(curl -s -H "Accept: application/vnd.github.v3+json" -H "Authorization: Basic $auth" "https://api.github.com/notifications?per_page=50")
+count=$(echo "$response" | jq -r 'length')
+
+if [ 0 = $count ]; then
+	echo 'No unread notifications'
```
Comment (thomaspaulmann) [inline developer-utils/github-unread-notifications.sh:46]: Should we keep this shorter? Seems like it's repeating the command name.

**Review by thomaspaulmann — CHANGES_REQUESTED** [review]
I think we can improve the title and message of the command to reduce the repetitive content. I left some suggestions inline. Also, could you add a screenshot of the command to the PR? This makes it easier to review. We should have probably a template for PRs that outlines this

**Comment by crstauf** [comment]
> could you add a screenshot of the command to the PR?

@thomaspaulmann Here you are! Having a PR template is a great idea!

Screen Shot 2020-11-06 at 9 11 15 AM
Screen Shot 2020-11-06 at 9 11 32 AM
Screen Shot 2020-11-06 at 9 11 47 AM

**Comment by thomaspaulmann** [comment]
Let's remove the white border of the GitHub icon. You can use that one. @PitNikola, we should add support for light/dark theme icons for script commands to make them pretty.

Here is the icon (it's white ;-)): 
github

**Comment by crstauf** [comment]
> here is the icon

@thomaspaulmann Won't this disappear for anyone using the light theme?

**Comment by thomaspaulmann** [comment]
Unfortunately it will but looks better on dark theme. We'll add support for light/dark theme icons. Then we can address it across all commands in this repo.

**Comment by crstauf** [comment]
@thomaspaulmann Okay; will update once #32 is merged.

**Comment by thomaspaulmann** [comment]
Ok, so I guess we need to merge this one first then.

**Comment by crstauf** [comment]
> Ok, so I guess we need to merge this one first then.

Actually I was planning to merge `master` into this branch, but this works too.

**Pull request merged**